markdown/developers/release_checklist.md

```diff
@@ -35,6 +35,9 @@ subtitle: A step-by-step guide for releasing a nf-core pipeline
 7. Go to GitHub and [create a new release for your pipeline](https://help.github.com/en/articles/creating-releases)
    - NB: Use _exactly_ the same version as in the code (e.g. `1.0.0`) - **do not prefix with v** (e.g. not `v1.0.0`).
    - Optional: Also include your [nice code name](http://www.codenamegenerator.com/) in your pipeline release title (see above with `CHANGELOG.md`)
+     - Prefix = Metal
+     - Dictionary = Animals
+     - Suffix = Don't use a suffix
```

Comment:
I don't think we have such strict requirements (Sarek uses places in the Sarek Park, Eager uses prehistoric finding sites (if I remember correctly, crispseq uses the nextflow style $adjective.$female_scientist (my guess), cageseq uses $metal.$animal_typically_held_in_cages)

Reply:
```suggestion
For example releases in nf-core/rnaseq follow the pattern:
     - Prefix = Metal
     - Dictionary = Animals
     - Suffix = Don't use a suffix
```

Reply:
I like it as a place to be a little creative, like having spellnames in airrflow etc :D. Maybe we can link a list with what others do to inspire ideas and just have the code name generator as one of them?

Reply:
Prehistoric sites?! Excuse me?! In eager our release names are the 14th century Swabian league of cities! 

In funcscan we are going with terrible national cuisine, like German Rollmops and British Beans on Toast (ok that one is pretty good actually).

But yeah we shouldn't hard code this -I'm pretty sure the 'cool release name' is just a suggestion even? It's not required?

Reply:
See, I didn't go for a creative naming scheme because it wasn't in the guide. Maybe we should require creative names?

Reply:
Derailing: but rollmops is delicious!

Reply:
I make lists of release names in issues milestones? Maybe that could be suggested (if I understand your point)